src/flight/filter.h

```diff
@@ -58,6 +72,7 @@ typedef union {
   filter_lp_pt1 lp_pt1;
   filter_lp_pt2 lp_pt2;
   filter_lp_pt3 lp_pt3;
+  filter_lp_lulu lp_lulu;
```

Comment:
if we make it part of the generic filter implementation, `filter_init` and `filter_step` should also handle this case, making it unnecessary to add dedicated handling to the dterm and gyro files.